front/src/Components/ConsoleGlobalMessageManager/InputTextGlobalMessage.svelte

```diff
@@ -35,19 +34,39 @@
     export let game: Game;
     export let gameManager: GameManager;
 
-    let gameScene = gameManager.getCurrentGameScene(game.scene.getScene(LoginSceneName));
+    let gameScene = gameManager.getCurrentGameScene(game.findAnyScene());
     let quill: Quill;
     let INPUT_CONSOLE_MESSAGE: HTMLDivElement;
 
     const MESSAGE_TYPE = AdminMessageEventTypes.admin;
 
+    export const handleSending = {
+        sendTextMessage() {
+            if (gameScene == undefined) {
+                return;
+            }
+            const text = quill.getText(0, quill.getLength());
+
+            const GlobalMessage: PlayGlobalMessageInterface = {
```

Comment:
InputTextGlobalMessage.svelte have responsibility to create and send messages? I prefer that it will be implemented in services or providers create for this. What do you think?

Reply:
I'm not 100% sure adding a service adds a lot of value here.
I mean, the service only purpose would be to call the `RoomConnection.emitGlobalMessage` function and I don't think functions that just call one function add a lot of value. If it was doing more (like checking rights, etc...), I'd be more willing to use a service but I'm not shocked if we skip using a service here.